### PR TITLE
Bump to version 2.5.0

### PR DESCRIPTION
**Pull request by n-rodriguez**
Hi! This patch contains the last version of FullCalendar (2.5.0).

### CODE REVIEW

File: lib/fullcalendar-rails/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Fullcalendar
   module Rails
-    VERSION = "2.4.0.2"
+    VERSION = "2.5.0"
```
Comment (davidwessman) [inline lib/fullcalendar-rails/version.rb]: Wrong format on this one.
https://github.com/bokmann/fullcalendar-rails/blob/2.4.0.0/lib/fullcalendar-rails/version.rb for reference.

**Comment by davidwessman** [comment]
:+1:  
Nice with people keeping it up to date :)

**Comment by n-rodriguez** [comment]
You're welcome!

**Comment by gr8bit** [comment]
Merged and gem pushed to rubygems. Thank you Nicolas!

**Comment by n-rodriguez** [comment]
Thanks to you for the fast integration/release :)

**Pull request merged**